x-pack/plugins/apm/public/services/rest/createCallApmApi.ts

```diff
@@ -10,12 +10,14 @@ import * as t from 'io-ts';
 import type {
   ClientRequestParamsOf,
   EndpointOf,
+  formatRequest as formatRequestType,
   ReturnOf,
   RouteRepositoryClient,
   ServerRouteRepository,
   ServerRoute,
 } from '@kbn/server-route-repository';
-import { formatRequest } from '@kbn/server-route-repository/target/format_request';
+// @ts-expect-error
```

Comment:
can you leave a comment as to what kind of error this is?

Reply:
done

Reply:
@dgieselaar are we okay to move on here?

Reply:
👍